modules/rosa-hcp-sts-creating-a-cluster-cli.adoc

```diff
@@ -0,0 +1,89 @@
+// Module included in the following assemblies:
+//
+// * rosa_getting_started_sts/rosa_creating_a_cluster_with_sts/rosa-sts-creating-a-cluster-quickly.adoc
+// * rosa_getting_started/rosa-getting-started.adoc
+// * rosa_getting_started/quickstart.adoc
+
+:_content-type: PROCEDURE
+[id="rosa-hcp-sts-creating-a-cluster-cli_{context}"]
+= Creating {hcp} for ROSA cluster using the CLI
+
+When using {hcp} for {product-title} (ROSA) CLI (`rosa`) to create a cluster, you can select the default options to create the cluster quickly.
+
+.Prerequisites
+
+* You have completed the AWS prerequisites for {hcp} for ROSA.
+* You have available AWS service quotas.
+* You have enabled the ROSA service in the AWS Console.
+* You have installed and configured the latest ROSA CLI (`rosa`) on your installation host.
++
+[NOTE]
+====
+To successfully install ROSA clusters, use the latest version of the ROSA CLI (`rosa`).
+====
+* You have logged in to your Red Hat account by using the `rosa` CLI.
+* You have verified that the AWS Elastic Load Balancing (ELB) service role exists in your AWS account.
+
+.Procedure
+
+//. Create environmental variables for your region and cluster name.
+//+
+//[source,terminal]
+//----
+//CLUSTER_NAME="<cluster-name>"
+//REGION="<region>"
+//----
+
+. You can create your {hcp} for ROSA cluster with one of the following commands. 
++
+[NOTE]
+====
+If you are using your own OIDC provider, you must include the endpoint URL and ARN arguments, such as `--oidc-endpoint-url <oidc_endpoint_url> --oidc-private-key-secret-arn <oidc_private_key_secret_arn>`.
+====
+
+.. Create a cluster with a single, initial machine pool, publicly available API, and publicly available Ingress by running the following command:
++
+[source,terminal]
+----
+$ rosa create cluster --cluster-name=<cluster_name> --sts --mode=auto --hosted-cp --subnet-ids=<public-subnet-id>,<private-subnet-id>
+----
+
+.. Create a cluster with a single, initial machine pool, privately available API, and privately available Ingress by running the following command:
++
+[source,terminal]
+----
+$ rosa create cluster --private --cluster-name=<cluster_name> --sts --mode=auto --hosted-cp --subnet-ids=<private-subnet-id>
+----
++
+[NOTE]
+====
+When you specify `--mode auto`, the `rosa create cluster` command creates the cluster-specific Operator IAM roles and the OIDC provider automatically. The Operators use the OIDC provider to authenticate.
+====
```

Comment:
IIUC, they will only do one of these, not both in sequence. If that's right, should be unordered bullets:
```suggestion
** Create a cluster with a single, initial machine pool, publicly available API, and publicly available Ingress by running the following command:
+
[source,terminal]
----
$ rosa create cluster --cluster-name=<cluster_name> --sts --mode=auto --hosted-cp --subnet-ids=<public-subnet-id>,<private-subnet-id>
----

** Create a cluster with a single, initial machine pool, privately available API, and privately available Ingress by running the following command:
+
[source,terminal]
----
$ rosa create cluster --private --cluster-name=<cluster_name> --sts --mode=auto --hosted-cp --subnet-ids=<private-subnet-id>
----
+
[NOTE]
====
When you specify `--mode auto`, the `rosa create cluster` command creates the cluster-specific Operator IAM roles and the OIDC provider automatically. The Operators use the OIDC provider to authenticate.
====
```